### PR TITLE
Copyright Headers - added to /web source files & linting step added

### PR DESCRIPTION
**Pull request by Cdok**
This PR updates the package.json to contain the new copyright
tool from the mobify-code-style repo. All copyright headers have been added to target `/web` source files.

Additionally, it adds a linting step for checking whether or not copyright headers are present in the targeted source files.

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-42
 **Linked PRs**: https://github.com/mobify/mobify-code-style/pull/155 and https://github.com/mobify/progressive-web-sdk/pull/622

## Changes
- Updates package.json
- Adds copyright headers to `'app/**/*.js?(x)' 'worker/**/*.js?(x)' 'tests/**/*.js?(x)'`

## How to test
- Checkout this branch
- Run `npm run copyright:lint`

## Todo
- [x] +1

### CODE REVIEW

File: web/app/static/js/jquery.min.js [hunk 1]
```diff
@@ -1,3 +1,7 @@
+/* * *  *  * *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  * */
```
Comment (jeremywiebe) [inline web/app/static/js/jquery.min.js]: This shouldn't have our copyright on it as this is a vendored file.

File: web/app/vendor/astro-detect.js [hunk 2]
```diff
@@ -1 +1,5 @@
+/* * *  *  * *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  * */
```
Comment (jeremywiebe) [inline web/app/vendor/astro-detect.js]: We should exclude anything in `web/app/vendor`  I know this file doesn't have a copyright header, so let's leave this addition here, but update the `package.json` to exclude this folder as the intention is that this folder contains 3rd party code that isn't under our copyright.

**Comment by Cdok** [comment]
@jeremywiebe accounted for those 2 directories in the new package.json. As soon as `2.8.1` is up on npm I'll re-run the tool

File: web/package.json [hunk 3]
```diff
@@ -109,9 +109,11 @@
     "build-sprites": "npm run clean:svgs && svg-sprite --symbol --symbol-dest='.' --symbol-sprite='sprite.svg' --shape-id-generator='pw-%s' --dest=./app/static/svg/sprite-dist ./app/static/svg/sprite-source/*.svg",
     "clean:svgs": "svgo --enable='removeUselessStrokeAndFill' -f ./app/static/svg/sprite-source",
     "clean:images": "imagemin-power -v --recursive --plugin=pngquant \"./app/static/img/**/*\" --out-dir=.",
+    "copyright:lint": "copyright 'app/!(vendor|static)/**/*.js?(x)' 'worker/**/*.js?(x)' 'tests/**/*.js?(x)'",
```
Comment (jeremywiebe) [inline web/package.json]: Oops, I missed a couple of other dirs/files.  
`/service-worker-loader.js`
`/webpack`

**Pull request merged**